src/docfx/build/context/BuildScope.cs

```diff
@@ -27,19 +25,23 @@ internal class BuildScope
 
         public RedirectionMap Redirections { get; }
 
+        public Docset Docset { get; }
+
+        public Docset FallbackDocset { get; }
```

Comment:
I try to hide FallbackDocset from outside of build scope, can we still keep it hidden?